.github/workflows/ci.yml

```diff
@@ -809,124 +809,15 @@ jobs:
     name: Check unreleased changes
     runs-on: ubuntu-latest
     needs: [tests, checkgovernance]
-    if: ${{ needs.checkgovernance.outputs.can_tag == '1' && (github.event_name == 'push' ||  github.event_name == 'schedule' ||  github.event_name == 'workflow_dispatch') }}
```

Comment:
Keep this if clause - we shouldn't be checking and of this if governance fails or if it's not a PR.
Arguably we shouldn't even be checking governance on a PR but that's not in scope for this PR.

Reply:
Opps.  Added back in